superset-frontend/src/explore/components/DatasourcePanel.tsx

```diff
@@ -131,7 +131,8 @@ const DatasourceContainer = styled.div`
     color: ${({ theme }) => theme.colors.grayscale.light1};
   }
   .form-control.input-sm {
-    margin-bottom: 0;
+    width: 96%;
```

Comment:
This 96% jumped out at me as a little funny... is there supposed to be some number of pixels on each side of the element? Maybe something like `width: calc(100% - ${({ theme }) => theme.gridUnit * 2}px);` would make sense (or whatever the gutter measurement should be?)

Reply:
yep!